db/migrate/20161021104336_create_authorized_user.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAuthorizedUser < ActiveRecord::Migration
```

Comment:
This migration is not needed because only the first admin needs to be created through migration.